composer.json

```
@@ -1,7 +1,7 @@
 {
 	"name": "livechat/module-magento-livechat",
 	"require": {
-	"php": "~5.5.0|~5.6.0|~7.0.0|~7.1.0|~7.2.0|~7.3.0|~7.4.0|~8.1.0|~8.2.0",
+		"php": "~5.5.0|~5.6.0|^7.0.0|^8.1.0",
```

```suggestion
		"php": "~5.5.0|~5.6.0|^7.0.0|^8.0.0",
```

Should we test it with 8.0 and allow it here?